bn-models/src/models/ashop/ashop-cruise.ts

```diff
@@ -13,13 +13,15 @@ interface NonFishingDay {
   date?: BoatnetDate;
   reason?: string;
   location?: LocationEvent;
+  tripId?: string;
```

Comment:
Now thinking whether this should be tripNum since the observer won't know which trip is which based off tripId's. Or maybe we should have two fields, one for tripNum and the other for the tripId associated with it.